shot/src/main/scala/com/karumi/shot/ShotPlugin.scala

```diff
@@ -4,15 +4,25 @@ import com.karumi.shot.android.Adb
 import com.karumi.shot.domain.Config
 import com.karumi.shot.screenshots.{ScreenshotsComparator, ScreenshotsSaver}
 import com.karumi.shot.tasks.{
-  RemoveScreenshotsTask,
+  DownloadScreenshotsTask,
   ExecuteScreenshotTests,
-  DownloadScreenshotsTask
+  RemoveScreenshotsTask
 }
 import com.karumi.shot.ui.Console
+import org.gradle.api.artifacts.{
+  Dependency,
+  DependencyResolutionListener,
+  ResolvableDependencies
+}
 import org.gradle.api.{Plugin, Project}
+import org.gradle.tooling.GradleConnector
+import org.gradle.tooling.model.build.BuildEnvironment
 
 class ShotPlugin extends Plugin[Project] {
 
+  def GRADLE_MIN_MAJOR = 3
+  def GRADLE_MIN_MINOR = 4
```

Comment:
As this is a constant we should define it a spart of the class companion object.

```scala
object ShotPlugin {
    private val minGradleVersionSupportedMajorNumber = 3
    private val minGradleVersionSupportedMinorNumber = 4
}
class ShotPlugin extends Plugin[Project] {
    import ShotPlugin._
    ...
}
```

In scala the usage of uppercase constants value is not part of the language code style. I'd also like to find a different name for the constants but I can't find it 😄